.github/workflows/documentation.yml

```
@@ -166,5 +166,5 @@ jobs:
         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         AWS_REGION: 'us-east-1'   # optional: defaults to us-east-1
-        SOURCE_DIR: 'docs/website/'      # optional: defaults to entire repository
-        DEST_DIR: 'docs/python/'      # optional: defaults to entire repository
+        SOURCE_DIR: 'docs/build/html/'      # optional: defaults to entire repository
+        DEST_DIR: 'docs/python/V.${{ github.event.release.tag_name }}/'      # optional: defaults to entire repository
```

The tags already have the v...

```suggestion
        DEST_DIR: 'docs/python/${{ github.event.release.tag_name }}/'      # optional: defaults to entire repository
```